extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -135,8 +134,9 @@ private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
 
 		$metaData['language'] = $wgLanguageCode;
 		$isProduction = empty( $wgDevelEnvironment ) && empty( $wgStagingEnvironment );
+		$isPrivateWiki = WikiFactory::isWikiPrivate( $wgCityId ) || $wgIsPrivateWiki;
```

Comment:
why both? :) ❓

Reply:
`WikiFactory::isWikiPrivate` checks for access permissions set in `wgGroupPermissionsLocal` content.
`$wgIsPrivateWiki` is just a boolean variable